proposals/0029-cooperative-vector.md

```diff
@@ -602,92 +609,90 @@ The vector-matrix intrinsics are expected to be supported in all shader stages.
 
 #### D3D12 API Additions
 
-Note: The enums and structs need to be updated from the coop_vec name, once a
-new name for the feature is decided.
-
 ### Check Feature Support
 
 This feature requires calling CheckFeatureSupport(). Additional D3D12_FEATURE
 enum and corresponding D3D12_FEATURE_DATA* structs (listed below) are added to
-enable discovering the Cooperative Vector tier along with the datatype and
+enable discovering the Linear Algebra: Matrix Vector tier along with the datatype and
 interpretation combinations supported by new vector-matrix intrinsics.
 
 ```c++
 typedef enum D3D12_FEATURE {
     ...
-    // Contains cooperative vector tier.
+    // Contains Linear Algebra: Matrix Vector tier.
     // NN tbd when implemented
     D3D12_FEATURE_D3D12_OPTIONSNN;
-    D3D12_FEATURE_COOPERATIVE_VECTOR;
+    D3D12_FEATURE_LINEAR_ALGEBRA_MATRIX_VECTOR;
```

Comment:
I'll wait for your response on the earlier comments before adding new ones since they are in the same vein, but maybe this can be `D3D12_FEATURE_LINEAR_ALGEBRA`